src/ui/Pricing/SubscriptionSettings/SubscriptionCard/Card.svelte

```diff
@@ -55,29 +64,18 @@
     </svelte:element>
 
     {#if subaction && onSubactionClick}
-      <button class="btn-2 mrg-m mrg--l" on:click={onSubactionClick}>{subaction}</button>
+      <button class="btn-2 body-2" on:click={onSubactionClick}>{subaction}</button>
```

Comment:
Removal of `caption` breaks existing desktop UI. Please, make sure it's consistent with Figma design